doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_custom_pcb.rst

```diff
@@ -110,21 +117,25 @@ The presence of a ``bicr`` node in the application devicetree will automatically
 Flashing the BICR binary
 ========================
 
+##TODO
+
 After the |NCS| build system generates the BICR binary, you must flash this binary manually.
 The content of BICR should be loaded to the SoC only once and should not be erased nor modified unless the PCB layout changes.
 To manually flash the generated :file:`bicr.hex` file to the SoC, use nRF Util as follows::
 
-    nrfutil device program --options chip_erase_mode=ERASE_NONE --firmware bicr.hex` --core Secure --serial-number <serial_number>
+    nrfutil device program --options chip_erase_mode=ERASE_NONE --firmware bicr.hex` --core Application --serial-number <serial_number>
 
 You need to follow this flashing process only one time, as the PCB configuration will not change.
 
```

Comment:
I think you can probably add another section here to validate the BICR binary at this stage already.

Currently the "best" way to do this is just to reset this device and see if it still works.
There are some additional validation steps that we could take (and potentially integrate into the tools if needed), that will warn about some classes of errors. However, as many BICR errors are deadly to the HW, the fact you can still work with the device afterwards indicates that the BICR is at least partially okay.

Reply:
One slightly ugly way to validate that the BICR is installed and was successfully run is to do this:
```
(venv) inry@inry-linux:~/proto/razer_bicr$ nrfutil device x-read --address 0x2F88FF1C --serial-num
ber 1051164514 --core Secure
1051164514
0x2F88FF1C: 289CFB73                              |s..(|
```
Values can be decoded like this:
```
        0x289CFB73: BICR was loaded without error.
        0xCE68C97C: There was a failure loading the BICR. Note: you would typically not see this value, as the chip would be dead...
        0xD78213DF: No BICR was loaded (as none could be found during bootup).
```
This can only be done in LCS Empty

Reply:
I would suggest more along these lines:

After programming the BICR binary onto the device, you should ensure that the BICR works with your device.
To do this perform the following steps:
* `nrfutil device reset --reset-kind RESET_PIN --serial-number <serial_number>`
* `nrfutil device x-read --address 0x2F88FF1C --serial-number 1051164514 --core Secure`

The second command has 4 plausible outcomes:
* Error - This indicates that the device is likely suffering from severe power issues after applying the BICR. This state is likely unrecoverable.
* ``0xCE68C97C``: BICR application failed. This indicates that there is an issue with the BICR, but in most cases this can be recovered by programming the correct BICR.
* ``0xD78213DF``: BICR application was skipped. Indicating that no BICR was programmed to the device.
* ``0x289CFB73``: BICR applied without error. This indicates that the power configuration of the BICR is valid and you can proceed to the next step.